Apps/Src/FaultState.c

```diff
@@ -0,0 +1,48 @@
+#include "FaultState.h"
+#include "Contactors.h"
+
+static void ArrayKill(void) {
+    Contactors_Set(ARRAY, OFF);
+}
+
+static void MotorKill(void) {
+    Contactors_Set(MOTOR, OFF);
+}
+
+void EnterFaultState(void) {
+    if(FaultBitmap.Fault_OS){
+        ArrayKill();
+        MotorKill();
```

Comment:
If you always call these together, why not just write a function that kills both?